app/static/js/createEvents.js

```diff
@@ -127,4 +129,39 @@ $(document).ready(function() {
       $("#checkIsRequired").prop('disabled', false);
     }
   });
+
+  var facilitatorArray = []
+  function callback(selectedFacilitator) {
+    // JSON.parse is required to de-stringify the search results into a dictionary.
+    let facilitator = (selectedFacilitator["firstName"]+" "+selectedFacilitator["lastName"]+" ("+selectedFacilitator["username"]+")");
+    let username = selectedFacilitator["username"];
+    if (!facilitatorArray.includes(username)){
+        facilitatorArray.push(username);
+        let tableBody = $("#facilitatorTable").find("tbody");
+        let lastRow = tableBody.find("tr:last");
+        let newRow = lastRow.clone();
+        newRow.find("td:eq(0) p").text(facilitator);
+        newRow.find("td:eq(0) div button").attr("data-id", username);
+        newRow.find("td:eq(0) div input").attr("id", username);
+        newRow.attr("id", username);
+        newRow.prop("hidden", false);
+        lastRow.after(newRow);
+        $("#hiddenFacilitatorArray").attr("value", facilitatorArray);
+    }
+  }
+
+  $("#eventFacilitator").on('input', function() {
+    // To retrieve specific columns into a dict, create a [] list and put columns inside
+    searchUser("eventFacilitator", callback, true, null, ["firstName", "lastName", "username"]);
```

Comment:
That last parameter isn't necessary anymore. Also, don't you need a category here?